src/esp/assets/ResourceManager.cpp

```diff
@@ -1508,6 +1527,9 @@ int ResourceManager::loadNavMeshVisualization(esp::nav::PathFinder& pathFinder,
   if (!pathFinder.isLoaded())
     return navMeshPrimitiveID;
 
+  if (!getCreateRenderer())
+    return navMeshPrimitiveID;
```

Comment:
should have "{}" even it is one line of code within the if clause based on our code convention.